modules/@apostrophecms/form-file-field-widget/index.js

```diff
@@ -3,6 +3,20 @@ module.exports = {
   options: {
     label: 'aposForm:file'
   },
+  fields: {
+    add: {
+      allowMultiple: {
+        label: 'aposForm:fileAllowMultiple',
+        type: 'boolean',
+        def: true
+      },
+      maxSize: {
```

Comment:
* Please put `maxSize` behind a `limitSize` boolean that hides it unless chosen using `if`, so it is clear that less technical form creators don't have to touch this, and check for `limitSize` being truthy before enforcing.